pilot/pkg/serviceregistry/kube/controller/namespacecontroller.go

```diff
@@ -76,7 +75,7 @@ func NewNamespaceController(data func() map[string]string, options Options, kube
 
 	watchedNamespaceList := strings.Split(options.WatchedNamespaces, ",")
 
-	mlw := listwatch.MultiNamespaceListerWatcher(watchedNamespaceList, func(namespace string) cache.ListerWatcher {
+	configmapmlw := listwatch.MultiNamespaceListerWatcher(watchedNamespaceList, func(namespace string) cache.ListerWatcher {
```

Comment:
This seems unrelated.

Reply:
Because I added another mlw, I simply made the name more descriptive.

Reply:
Sure. I was mostly thinking that it was an unnecessary diff to carry, but this has all vanished upstream anyway.